SU-HSDO/suhumsci
# STN-351: QA Feedback 16

# [STN-372](https://sparkbox.atlassian.net/browse/STN-351) READY FOR REVIEW

## Summary
Resolve issues discovered during the Sprint 16 QA Feedback session listed in STN-351

## Need Review By (Date)
5/26

## Urgency
medium

## Steps to Test
1. In the CLI run `npm run test` and confirm all of our Sass tests pass.
<br>

2. Image Styles on node pages
- [x] In the top section of the node detail page, images should maintain their size and aspect-ratio as defined by the image style. The image size should remain fixed and the text region should be flexible. Pages to review:
* http://economics.suhumsci.loc/events/qa-events-all-fields-filled
* http://economics.suhumsci.loc/news/qa-joeys-test-news-item
* http://economics.suhumsci.loc/people/qa-person-all-fields-filled
* http://economics.suhumsci.loc/events/gsb-organizational-behavior-seminars/gsb-organizational-behavior-seminars-13
<br>

3. Pills 
Find a page that is currently using pills or update a page such as http://economics.suhumsci.loc/people/faculty. 
* Update the view so that `Research Areas` Multiple Field Settings > Display Type uses an `ol` and in Style Settings > Customize field HTML add the class of `hb-pill-link-list`. 
Improve the color contrast of pills in Colorful
* Update `Fields of Interest` Multiple Field Settings > Display Type to use an `ul` and in Style Settings > Customize field HTML add the class of `hb-pill-list`.
- [x] Confirm that unordered lists do not display bullets before the pill. See screenshot in [STN-351](https://sparkbox.atlassian.net/browse/STN-351) for clarification.

- [x] Improve the color contrast of pills in Colorful
Review the colors of the pills in each color pairing (Ocean, Mountain, and Cardinal). Confirm the color contrast for each has improved and that they meet [WCAG AA](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) guidelines.
<br>

4. More link
- [x] Review more link when located in the local footer dark variant. Confirm color contrast improvement and that it meets WCAG AA guidelines.
<br>

5. Row Background Color
Create a page filled for testing content within rows that have full width and limited width wells. The the screenshot for `row background color` in JIRA STN-351 as an example. Create the following scenarios for full width and limited width wells:
- [x] Views without a title --> look for adequate spacing on top
- [x] horizontal card postcard --> look for adequate spacing on top
- [x] accordions --> Make sure they span the width of the container and the plus sign does not overlap the text. I also added a background-color to the accordion summary as it was the background-color of the well.
- [x] hero --> Confirm they span the width of the container. I also made spacing adjustments for consistency.
- [x] carousel --> I made spacing adjustments for consistency and changed the background color of the grey block behind the image because it was the same color as the well. 
<br>

6. Mixing bullets and numbers in WYSIWYG results
- [x] Go to https://economics-prod.stanford.edu/graduate/phd-degree and confirm that the bullets and numbers within the `General Timeline` lists do not overlap upon each other.
<br>

7. Date Stacked Horizontal Card
- [x] Go to http://economics.suhumsci.loc/seminars-workshops/upcoming-seminars-and-events and confirm that the date, location, and speaker icon display for highlighted events and upcoming seminars.
<br>

8. Mailto links on postcards render the envelope icon
- [x] This was fixed in QA Feedback Sprint 15 work. Confirm fix by going to https://economics-dev.stanford.edu/undergraduate/honors-program. Scroll towards the bottom of the page and look for the `How to Apply` postcard.
<br>

9. More Link Traditional Arrow not quite centered (it is a tad too high)
- [x] On the Traditional theme, confirm the alignment of the arrows on vertical linked cards and the more link has improved.
<br>

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
